tools/clang/test/CodeGenDXIL/hlsl/types/longvec-operators.hlsl

```diff
@@ -48,24 +48,6 @@ struct Interface {
   TYPE scales[10];
 };
 
-#if 0
-// Requires vector loading support. Enable when available.
```

Comment:
Since this is the change which introduces said support, why is this deleted instead of used?